lib/modules/manager/gomod/extract.ts

```diff
@@ -102,3 +97,29 @@ export function extractPackageFile(content: string): PackageFile | null {
   }
   return { constraints, deps };
 }
+
+function parseMultiLine(
+  startingLine: number,
+  lines: string[],
+  matchRegex: RegExp,
+  blockType: 'require' | 'replace'
+): { reachedLine: number; detectedDeps: PackageDependency[] } {
```

Comment:
please create an interface for type, can be private here

Reply:
Sure